config/manager/manager.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         runAsNonRoot: true
       containers:
         - command:
-            - /manager
+            - /csi-addons-manager
```

Comment:
same change should also be made to CSV? might be missed to updated CSV with git push

Reply:
Hmm, I don't think the command is included there. I can not find it in `config/manifests/bases/clusterserviceversion.yaml.in`.